### PR TITLE
add hide/show class for mode & owner

### PR DESCRIPTION
**Pull request by ericfranz**
The hide-owner-mode css class on the body (or above the panel) will hide these rows.
By default these are supposed to be hidden so I updated the body tag to
have this class. This is using flexbox completely now - no more arbitrary percent widths
on columns.

This should be toggle-able via a checkbox.

Fixes https://github.com/AweSim-OSC/osc-fileexplorer/issues/72

### CODE REVIEW

**Comment by brianmcmichael** [comment]
@ericfranz needs review.

I've pulled in the changes from the dotfiles branch, added the checkbox toggler and cookie handling and externalized the custom javascripts.

**Comment by brianmcmichael** [comment]
Something about this broke the tree click handler. Looking at it now.

File: html/index.html [hunk 1]
```diff
@@ -1,115 +1,86 @@
...
+CloudCmd('{{ prefix }}');
+
+// Event handler activates on page load
 jQuery12('body').on('panel_initialized', function(e){
     initTree("{{ treeroot }}");
+    document.getElementById("checkbox-dotfiles").checked = ( getCookie('dotfiles') === 'true' );
+    dotfiles();
+    document.getElementById("checkbox-ownermode").checked = ( getCookie('ownermode') === 'true' );
+    ownerMode();
 });
```
Comment (ericfranz) [inline html/index.html]: This should be before `CloudCmd('{{ prefix }}');` not after. In fact, the safest is before any of Cloud Commander's own code is pulled in:

``` html
<script src="{{ prefix }}/lib/client/ood_tree.js"></script>
<script src="{{ prefix }}/lib/client/ood.js"></script>
<script>
// Event handler activates on page load
jQuery12('body').on('panel_initialized', function(e){
    initTree("{{ treeroot }}");
    document.getElementById("checkbox-dotfiles").checked = ( getCookie('dotfiles') === 'true' );
    dotfiles();
});
</script>


<script src="{{ prefix }}/cloudcmd.js"></script>
<script>
CloudCmd('{{ prefix }}');
</script>
```

**Comment by ericfranz** [comment]
You put this in the <head>

```
<script src="{{ prefix }}/lib/client/ood_tree.js"></script>
```

It needs to be in the body because it adds click handlers to the body.

**Comment by brianmcmichael** [comment]
@ericfranz try it now.

**Pull request merged**